_data/projects/kentico-communitycontentbacklog.yml

```diff
@@ -0,0 +1,17 @@
+name: community-content-backlog
+desc: Kentico community content backlog.
+site: https://github.com/Kentico/community-content-backlog
+tags:
+- content-management
+- sdk
+- blog
+- bloggers
+- tutorial
+- kentico
+- kentico-cloud
+- cms
+- headless-cms
+- caas
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/Kentico/community-content-backlog/labels/up-for-grabs
```

Comment:
@Simply007  There aren't any issues labeled with this tag.

Reply:
@ritwik12 - sorry about that - I  forgot to label the issues in the project - it should be labeled now!